vhdbuilder/packer/install-dependencies.sh

```diff
@@ -457,7 +460,9 @@ EOF
 
 fi
 
-ls -ltr /opt/gpu/* >> ${VHD_LOGS_FILEPATH}
+if [ -d "/opt/gpu" ] && [ "$(ls -A /opt/gpu)" ]; then
```

Comment:
why isn't `[ -d "/opt/gpu" ]` enough here?

Reply:
I tested this. If the folder exists but empty, next line `ls -ltr` will return error. So we need the second check.

Reply:
❯ mkdir test
❯ ls -ltr test
total 0
❯ echo $?
0

wierd, I dont get an error code

❯ ls -ltr test/*
zsh: no matches found: test/*
❯ echo $?
1

the * does it